openssl-dynamic/src/main/native-package/configure.ac

```diff
@@ -28,7 +28,7 @@ AC_CONFIG_HEADERS([src/config.h])
 AC_CANONICAL_HOST
 AC_CANONICAL_SYSTEM
 
-${CFLAGS="-O3 -Werror -fno-omit-frame-pointer"}
+${CFLAGS="-O3 -Werror -fno-omit-frame-pointer -Wunused-variable"}
```

Comment:
If you feel extra adventurous later on, `-Wall` is fun. (`-Wall` implies `-Wunused-variable` so you won't need both.)